projects/ad7124-4sdz/src/ad7124-4sdz.c

```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
+#include <inttypes.h>
+#include "ad7124.h"
+#include "ad7124_regs.h"
+#include "spi_extra.h"
+
+/******************************************************************************/
+/********************** Macros and Constants Definitions **********************/
+/******************************************************************************/
+
+int main(void)
+{
+	struct ad7124_dev *dev;		/* A driver handle to pass around */
```

Comment:
nitpick: can you indent variable names. This way is easier to differentiate name from type